app/src/main/res/navigation/public_share_navigation.xml

```diff
@@ -33,13 +33,22 @@
             app:argType="string" />
         <argument
             android:name="fileId"
+            android:defaultValue="-1"
```

Comment:
It seems we don't use that -1 value ?

Reply:
We need it when navigating to the activity in the case of needed password